src/Department/Department.js

```diff
@@ -11,29 +11,29 @@ class Department extends Component {
   }
 
   componentDidMount () {
-    const departmentid = this.props.match.params.id;
-    const getDepartamentById = async () => {
-      const data = await API.getDepartamentById(departmentid);
-      if (data.statusText === "OK") {
-        this.setState({
-          departmentDescription: data.data.description
-        })
-      } else {
-        this.setState({
-          requestError: data
-        })
-      }
-    };
+    this.getDepartamentById()
+  }
 
-    getDepartamentById();
+  getDepartamentById = async () => {
+    const { match: { params: { id } } } = this.props
```

Comment:
it is better to use ?. there instead of destructuring.

Reply:
but ESLint won't let me do it, asks to do destructing

Reply:
it will cause error if params doesn't exist